pyproject.toml

```diff
@@ -137,14 +137,9 @@ addopts = [
   "-ra",
   "--strict-config",
   "--strict-markers",
-  "--doctest-modules",
```

Comment:
It is a good idea to have this. Better to remove bad doctests than have them without running them.